sqlx-core/src/types/mod.rs

```diff
@@ -46,6 +46,15 @@ pub mod chrono {
     };
 }
 
+#[cfg(feature = "jiff")]
+#[cfg_attr(docsrs, doc(cfg(feature = "jiff")))]
+pub mod jiff {
+    #[doc(no_inline)]
+    pub use jiff::{
+        Zoned, Timestamp, tz::TimeZone, tz::Offset, civil::Date, civil::Time, civil::DateTime,
```

Comment:
Determinate which type should be used. Roughtly,

* chrono::Duration -> jiff::SignedDuration
* chrono::NativeDate -> jiff::civil::Date
* chrono::NativeTime -> jiff::civil:Time
* chrono::NaiveDateTime -> jiff::Timestamp
* chrono::DateTime -> Zoned

Reply:
chrono::NaiveDateTime seems should be jiff::civil::DateTime as it's without timezone.

Reply:
Yes, `chrono::NaiveDateTime` should be `jiff::civil::DateTime`.